code/modules/admin/player_panel.dm

```diff
@@ -326,7 +326,7 @@
 		//dat += "</div>"
 
 	//Coder options
```

Comment:
could remove this comment

Reply:
Uh, too late. I'll retroactively have left it in there for posterity.